conda-build/common/delete_old_night_packages.py

```diff
@@ -9,7 +9,10 @@
 import subprocess
 import sys
 
-MAX_NUMBER_OF_PACKAGES = 30
+# We currently can't have any retention as our average binary
+# size is ~150 MB, so even 1 day of retention would require
+# storing 24 binaries, which is over the 3 GB limit
+MAX_NUMBER_OF_PACKAGES = 0
```

Comment:
Change to 6 for now.

Reply:
A number that isn't an integer multiple of the number of packages we upload seems strange as we are just keeping an arbitrary selection

Reply:
Typo: Meant 8

Reply:
This feels like we are just leaving ourselves a very hard to catch bug for once we fix the headless == false bullet == true build

Reply:
It actually is by upload date (well version).  A package according to this name is a version.  I changed the name to be less confusing.  I'll make a version that does the delete right before uploading everything and the new can do no retention later, but that is way more involved while just retaining 1 version can be pushed tonight